shell/packages/sandstorm-db/db.js

```diff
@@ -582,6 +582,12 @@ matchWildcardHost = function(host) {
 
   var prefix = wildcardHost[0];
   var suffix = wildcardHost[1];
+
+  // We ignore everything after the first : character to be agnostic
```

Comment:
I think my plan is to actively break that use-case. Here's my rationale:

* `nginx` by default (in my testing on my laptop, with nginx on `:80` and a Flask app on `:5000`) when I use `proxy_pass`, sends a `Host` header of `localhost:5000` because it's making an HTTP request to `localhost:5000`.
* You can configure a line of `    proxy_set_header Host $host;` and that sends `Host: localhost`.
* If you try really hard, it's probably possible to get nginx to send `Host localhost:80` but I guess my point is that's a pretty rare configuration and probably zero users are using that configuration.

So I think I'm going to break this, and if someone complains, then I'll figure out what to do. Maybe that means we should blog a short note about it, or something, or I should send a note to sandstorm-dev about the breaking change.